ffc-pay-alerts/schema/event-schema.js

```diff
@@ -0,0 +1,20 @@
+module.exports = [{

```

Comment:
just a consideration but could this be extracted to `local.settings.json` or part of it, perhaps `notifyTemplateId`?

what was your reason for defining it as a module?

Reply:
There may be need to add more event-templates, also, we may need to include other properties, hence, if added to `local.settings.json`, it may become convoluted as we add more entries and/or properties.